DrawerKit/DrawerKit/Internal API/PresentationController.swift

```diff
@@ -54,6 +86,10 @@ extension PresentationController {
     }
 
     override func presentationTransitionWillBegin() {
+        // NOTE: `targetDrawerState.didSet` is not invoked within the
```

Comment:
That's supposed to be common knowledge (that the property observers don't get triggered inside initialisers) so I don't think we need the comment here.